docs: changelog-2024-12-13 (#2631)
neondatabase/website: content/changelog/2024-12-13.md

---
description: Browser-based psql client, larger compute sizes, and project-scoped API keys
---

> ## Introducing psql.sh — run psql directlyin your browser

Reviewer: ```suggestion
## Introducing psql.sh: run psql directly in your browser
```

We're excited to launch [psql.sh](https://psql.sh), a browser-based version of the PostgreSQL command-line client. This new tool lets you instantly spin up a fresh database and start running SQL queries and psql commands right in your browser.

![psql.sh interface showing the terminal emulator](/docs/relnotes/psql_sh.png)

> psql.sh uses Neon's branching to create instant, isolated database environments for you to experiment with. Try it out at [psql.sh](https://psql.sh), or read about how we built it in our blog post [psql from the browserhttps://neon.tech/blog/psql-from-the-browser.

Author: ```suggestion
psql.sh uses Neon's branching to create instant, isolated database environments for you to experiment with. Try it out at [psql.sh](https://psql.sh), or read about how we built it in our blog post [psql from the browser](https://neon.tech/blog/psql-from-the-browser).
```

---
description: Larger compute sizes, project-scoped API keys, and a browser-based psql client
---

## Larger compute sizes are here

We’ve added support for larger compute sizes:

- Neon's autoscaling feature now supports a maximum setting of **16 vCPU**. Compute sizes larger than 8 vCPU are available on our Business and Enterprise plans.
- Neon's Business and Enterprise plans now support fixed compute sizes up to **56 vCPU**. Autoscaling is not yet supported for compute sizes larger than 16 vCPU but [reach out to us](https://neon.tech/contact-sales) if you'd like to extend your limit beyond that.

## Project-scoped API keys

We now support per-project API keys, offering you finer-grained access control by limiting an API key's permissions to a particular project. This applies to organization-owned projects only — when creating an Organization API key, include the project ID in the request and the resulting key will be scoped to just that project. This key grants member-level permissions only, preventing destructive actions like deleting the project.

> 

Reviewer: ```suggestion
```

---

> description: Browser-based psql client, larger compute sizes, and project-scoped API keys

Author: ```suggestion
description:  Larger compute sizes, project-scoped API keys, and a browser-based psql client
```

---

## Larger compute sizes are here

We’ve added support for larger compute sizes:

- Neon's autoscaling feature now supports a maximum setting of **16 vCPU**. Compute sizes larger than 8 vCPU are available on our Business and Enterprise plans.

> - Neon's Business and Enterprise plans now support fixed compute sizes up to **56 vCPU**. Autoscaling is not yet supported for compute sizes larger than 16 vCPU but [reach out to us](https://console.neon.tech/app/projects?modal=support) if you'd like to extend your limit beyond that.

Reviewer: ```suggestion
- Neon's Business and Enterprise plans now support fixed compute sizes up to **56 vCPU**. Autoscaling is not yet supported for compute sizes larger than 16 vCPU but [reach out to us](https://neon.tech/contact-sales) if you'd like to extend your limit beyond that.
```

## Project-scoped API keys

We now support per-project API keys, offering you finer-grained access control by limiting an API key's permissions to a particular project. This applies to organization-owned projects only — when creating an Organization API key, include the project ID in the request and the resulting key will be scoped to just that project. This key grants member-level permissions only, preventing destructive actions like deleting the project.

> For more detail, see [Project-scoped API keys](/docs/manage/api-keys#project-scoped-api-keys).

Reviewer: ```suggestion
```

We’ve added support for larger compute sizes:

- Neon's autoscaling feature now supports a maximum setting of **16 vCPUs**. For configurations with a maximum limit of 10 vCPUs or higher, the maximum cannot exceed 8 times the minimum setting. For example, if your minimum is set to 2 vCPUs, your maximum can't exceed 16 vCPUs.
- Neon's Business and Enterprise plans now support fixed compute sizes up to **56 vCPUs**. Autoscaling is not supported for compute sizes larger than 16 vCPU.

## Project-scoped API keys

> We now support per-project API keys, offering you finer-grained access control by limiting an API key's permissions to a particular project. This applies to organization-owned projects only — when creating an Organization API key, include the project ID in the request and the resulting key will be scoped to just that project. This key grantsmember-level permissions only, preventing destructive actions like deleting the project.

Reviewer: ```suggestion
We now support per-project API keys, offering you finer-grained access control by limiting an API key's permissions to a particular project. This applies to organization-owned projects only — when creating an Organization API key, include the project ID in the request and the resulting key will be scoped to just that project. This key grants member-level permissions only, preventing destructive actions like deleting the project.
```
